crates/solidity/inputs/language/bindings/rules.msgb

```diff
@@ -1331,13 +1331,19 @@ inherit .star_extension
 ;; @id_path.rightmost_identifier which corresponds to the identifier in the last
 ;; position in the path, from left to right. This is used in the using directive
 ;; rules to be able to pop the name of the attached function.
+;;
```

Comment:
Is there a case of the form X.Y.z where this can arise? (More than one period). Is this handled?